<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="20407" started-at="2021-06-24T16:11:00 IST" finished-at="2021-06-24T16:11:21 IST">
    <groups>
    </groups>
    <test name="Default test" duration-ms="20407" started-at="2021-06-24T16:11:00 IST" finished-at="2021-06-24T16:11:21 IST">
      <class name="TestCases.LoginTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:TestCases.LoginTest@3043fe0e]" name="setUp" is-config="true" duration-ms="10126" started-at="2021-06-24T16:11:00 IST" finished-at="2021-06-24T16:11:11 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="openPage()[pri:0, instance:TestCases.LoginTest@3043fe0e]" name="openPage" is-config="true" duration-ms="3775" started-at="2021-06-24T16:11:11 IST" finished-at="2021-06-24T16:11:14 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openPage -->
        <test-method status="FAIL" signature="verifyLoginWithValidCred()[pri:0, instance:TestCases.LoginTest@3043fe0e]" name="verifyLoginWithValidCred" duration-ms="4639" started-at="2021-06-24T16:11:14 IST" finished-at="2021-06-24T16:11:19 IST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [false] but found [true]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertFalse(Assert.java:67)
at org.testng.Assert.assertFalse(Assert.java:77)
at pages.LoginPage.verifySuccesfullLogin(LoginPage.java:36)
at TestCases.LoginTest.verifyLoginWithValidCred(LoginTest.java:33)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLoginWithValidCred -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:TestCases.LoginTest@3043fe0e]" name="tearDown" is-config="true" duration-ms="1291" started-at="2021-06-24T16:11:19 IST" finished-at="2021-06-24T16:11:21 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- TestCases.LoginTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
